modules/snowflake-account/versions.tf

```
@@ -0,0 +1,18 @@
+terraform {
+  required_version = ">= 1.0.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+    snowflake = {
+      source  = "chanzuckerberg/snowflake"
+      version = "~> 0.25.36"
```

Use of the pessimistic operator on the patch release is too specific

```suggestion
      version = "~> 0.25"
```